### PR TITLE
Basic UI shims for cross-network

### PR DESCRIPTION
**Pull request by NiloCK**
image

PR also includes spots for chain metadata. Notably block explorer and exchange rates.

### CODE REVIEW

File: src/chains.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+   * The value of the chain's native token in USD. Permits cross-chain exchange rate calculations.
+   */
+  exchangeRate: number;
+}
+
+export const chains: ChainData[] = [
+  {
+    url: "http://localhost:8545",
+    chainId: 31337,
+    name: "hardhat 1",
```
Comment (lalexgap) [inline src/chains.ts:26]:
```suggestion
    name: "hardhat A",
```

File: src/chains.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    symbol: "hh1ETH",
```
Comment (lalexgap) [inline src/chains.ts:27]:
```suggestion
    symbol: "hhAETH",
```

File: src/chains.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    explorer: "",
+    exchangeRate: 1,
+  },
+  {
+    url: "http://localhost:8546",
+    chainId: 31338,
+    name: "hardhat 2",
```
Comment (lalexgap) [inline src/chains.ts:34]:
```suggestion
    name: "hardhat B",
```

File: src/chains.ts [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  /**
+   * The value of the chain's native token in USD. Permits cross-chain exchange rate calculations.
+   */
+  exchangeRate: number;
+}
+
+export const chains: ChainData[] = [
+  {
+    url: "http://localhost:8545",
+    chainId: 31337,
```
[the remark this answers is not in the document]
Comment (NiloCK) [reply src/chains.ts:25]: I think "going to prod" (going to testnet) may require it if we want the L1Pay button to still do something.

[the remark this answers is not in the document]
Comment (NiloCK) [reply src/chains.ts:25]: eg, the UserOperation that Irene submits will need to be wrapped in a L1tx that has the correct chainID

[the remark this answers is not in the document]
Comment (lalexgap) [reply src/chains.ts:25]: > eg, the UserOperation that Irene submits will need to be wrapped in a L1tx that has the correct chainID

Irene's client can already get the [correct chain id](https://github.com/statechannels/SCBridge-Wallet/blob/ef11e9f6c70796db08b68ca9076c733b94152a9d/clients/StateChannelWallet.ts#L237) through the provider. I don't think we need to also add a hardcoded chain id?

**Review by lalexgap — APPROVED** [review]

File: src/chains.ts (hunk 4 above)
[the remark this answers is not in the document]
Comment (NiloCK) [reply src/chains.ts:25]: Very good! The provider takes the URL only, and takes care of the rest?

Will snip.

**Comment by lalexgap** [comment]
Since I merged https://github.com/statechannels/SCBridge-Wallet/pull/136 I ended up re-targetting and rebasing this off of `main`.

**Pull request merged**